code.gs

```diff
@@ -156,6 +168,13 @@ var settings = {
      * Choose a date you know should trigger an event notification.
      */
     testDate: new Date('2017/10/19 06:00:00')
+  },
+  developer: {
+    /* NB: Users shouldn't need to (or want to) touch these settings. They are here for the
+     *     convenience of developers/maintainers only.
+     */
+    version: '3.2.0-alpha',
```

Comment:
Moving the version number here would require [line R1173](https://github.com/GioBonvi/GoogleContactsEventsNotifier/pull/60/files#diff-deb2c49f6ef99255c9830fe80e966b46R1173) to be updated with a new RegEx.

However, since changing the RegEx would break older version update detection and this is not the first time that this happens I think we should be using [GitHub's API](https://developer.github.com/v3/repos/releases/#get-the-latest-release) to detect the latest released version of the script, since it would be more reliable (no RegEx, just JSON parsing), more stable (no changes breaking retrocompatibility) and would still let use move the _current version number_ around as we please.

Let me know if you want to add this to the PR in a new commit, otherwise I'll do it when this PR is merged.